utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIast.java

```diff
@@ -299,6 +300,18 @@ void secureXSS(final ServletResponse response) throws IOException {
         xssExamples.secureXSS(response.getWriter());
     }
 
+    @GetMapping(value = "/hstsmissing/test_insecure", produces = "text/html")
+    public String hstsHeaderMissingInsecure(HttpServletResponse response) {
+      response.setStatus(HttpStatus.OK.value());
+      return "ok";
+    }
+
+    @GetMapping(value = "/hstsmissing/test_secure", produces = "image/gif")
+    public String hstsHeaderMissingSecure(HttpServletResponse response) {
```

Comment:
The secure case should be using text/html and setting the appropriate headers.

Reply:
done

Reply:
change applied